src/sentry/release_health/metrics.py

```diff
@@ -81,28 +82,28 @@
 # In order to make functional comparison easier, the metrics implementation
 # (explicitly) chooses the same rollup in the equivalent queries, and uses this
 # constant to denote that case.
-LEGACY_SESSIONS_DEFAULT_ROLLUP = 3600
+MINUTE = 60  # 60 seconds
+HOUR = MINUTE * 60
+DAY = HOUR * 24
+LEGACY_SESSIONS_DEFAULT_ROLLUP = HOUR
 USE_CASE_ID = UseCaseKey.RELEASE_HEALTH
 
 logger = logging.getLogger(__name__)
 
-
 _K1 = TypeVar("_K1")
 _K2 = TypeVar("_K2")
 _V = TypeVar("_V")
 
 
 def filter_projects_by_project_release(project_releases: Sequence[ProjectRelease]) -> Condition:
-    return Condition(Column("project_id"), Op.IN, list(x for x, _ in project_releases))
+    return Condition(Column("project_id"), Op.IN, [proj for proj, _rel in project_releases])
 
 
-def filter_releases_by_project_release(
-    org_id: int, project_releases: Sequence[ProjectRelease]
-) -> Condition:
+def filter_releases_by_project_release(project_releases: Sequence[ProjectRelease]) -> Condition:
     return Condition(
-        Column(resolve_tag_key(USE_CASE_ID, org_id, "release")),
-        Op.IN,
-        resolve_many_weak(USE_CASE_ID, org_id, [x for _, x in project_releases]),
+        lhs=Column(name="tags[release]"),
```

Comment:
i'm not familiar with this "tags[release]", may be @untitaker can help, do we want to avoid USE_CASE_ID here?

Reply:
solved in discussion offline.